classes/submission/SubmissionDAO.inc.php

```diff
@@ -452,6 +465,114 @@ function getByStatus($status, $stageUserId = null, $reviewUserId, $contextId = n
 	}
 
 	/**
+	 * Get all submissions that are considered assigned to the passed user.
+	 * @param $userId int
+	 * @param $contextId int optional
+	 * @param $title string|null optional Filter by title.
+	 * @param $author string|null optional Filter by author.
+	 * @param $stageId int|null optional Filter by stage id.
+	 * @param $rangeInfo DBResultRange optional
+	 * @return DAOResultFactory
+	 */
+	function getAssignedToUser($userId, $contextId = null, $title = null, $author = null, $stageId = null, $rangeInfo = null) {
+		$params = $this->_getFetchParameters();
+		$userId = (int) $userId;
+		array_push($params, $userId, STATUS_DECLINED, $userId, $userId, ROLE_ID_AUTHOR, $userId);
+		if ($contextId) $params[] = $contextId;
+
+		if ($title) {
```

Comment:
What if the user is browsing in a locale that doesn't have a specific translation? That'll never be found. OTOH I think it's better to search titles in all languages, and if any match, show the submission.